Source/WTF/wtf/threads/Signals.h

```diff
@@ -174,16 +175,93 @@ using WTF::SignalHandler;
 using WTF::addSignalHandler;
 using WTF::activateSignalHandlersFor;
 
-#else // not OS(UNIX)
+#elif OS(WINDOWS) // not OS(UNIX)
+
+#include <windows.h>
 
 namespace WTF {
 
+enum class Signal {
+    FloatingPoint,
+    IllegalInstruction,
+    AccessFault,
+    NumberOfSignals = AccessFault + 1,
+    Unknown
+};
+
+inline Signal fromSystemException(int signal)
+{
+    switch (signal) {
+    case EXCEPTION_FLT_DENORMAL_OPERAND:
+    case EXCEPTION_FLT_DIVIDE_BY_ZERO:
+    case EXCEPTION_FLT_INEXACT_RESULT:
+    case EXCEPTION_FLT_INVALID_OPERATION:
+    case EXCEPTION_FLT_OVERFLOW:
+    case EXCEPTION_FLT_STACK_CHECK:
+    case EXCEPTION_FLT_UNDERFLOW:
+        return Signal::FloatingPoint;
+    case EXCEPTION_ILLEGAL_INSTRUCTION:
+        return Signal::IllegalInstruction;
+    case EXCEPTION_ACCESS_VIOLATION:
+        return Signal::AccessFault;
+    default:
+        return Signal::Unknown;
+    }
+}
```

Comment:
I know this is mirroring `fromSystemSignal` but looking at the codebase it seems like both it and `toSystemSignal` aren't used outside of the `Signals.cpp` file so they both can be moved in there.

For the windows bits it saves us an include of `windows.h` if you just move it into the .cpp file